rwlib/CMakeLists.txt

```diff
@@ -41,10 +41,9 @@ SET(RWLIB_SOURCES
 	"source/job/WorkContext.cpp"
 	)
 
-add_library(rwlib
+set_source_files_properties(source/gl/gl_core_3_3.c PROPERTIES COMPILE_FLAGS -w)
```

Comment:
Instead, aren't we able to somehow generate a better source file to begin with?
If this is an issue for us, it's probably for others too. Has this been fixed in glLoadGen since?